src/client/src/apps/MainRoute/widgets/spotTile/components/Tile/TileBusinessLogic.ts

```diff
@@ -120,7 +120,12 @@ export const getDerivedStateFromUserInput = ({
   const { symbol } = currencyPair
   const { rfqState } = spotTileData
 
-  const notional = !isEditMode(value) ? getFormattedValue(value) : value
+  let notional: string
+  if (type === 'keypress' && isEditMode(value)) {
+    notional = getFormattedValue(value)
+  } else {
+    notional = !isEditMode(value) ? getFormattedValue(value) : value
+  }
```

Comment:
Looks like you could achieve the same result changing line 147
from `if (type === 'blur' && isEditMode(notional))`
to `if ((type === 'blur' || type === 'keypress') && isEditMode(notional))`

Reply:
@fdore you're right. Refactored the logic to this, thanks.